### DISCUSSION TITLE
table 支持 dark 模式吗?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#43846] [Q&A] [answered]
dark 模式好像只有少数几个组件支持,是吗? 比如 menu
https://ant.design/components/menu-cn#components-menu-demo-theme

**Comment by MadCcc** [comment, marked as answer]
Try
```tsx
import { theme, ConfigProvider, Table } from 'antd';

export default () => (
  <ConfigProvider theme={{ algorithm: theme.darkAlgorithm }}>
    <Table />
  </ConfigProvider>
);
```

**Reply by vaynevayne** [reply]
可以用
不过这个翻页组件的总有图标可能没适配吧,这个能帮忙看看吗?

<img width="1234" alt="image" src="https://github.com/ant-design/ant-design/assets/124666577/f0daf416-0c7b-431b-ae31-9a1ccceeabaa">

**Reply by vaynevayne** [reply]
好了, 重新刷新了下
